tests/common/ixia/qos_fixtures.py

```diff
@@ -102,3 +102,19 @@ def lossy_prio_list(all_prio_list, lossless_prio_list):
     """
     result = [x for x in all_prio_list if x not in lossless_prio_list]
     return result
+
+@pytest.fixture(scope="function")
+def start_pfcwd_default(duthosts, rand_one_dut_hostname):
+    """
+    Start PFC watchdog with default setting
+
+    Args:
+       duthosts (pytest fixture) : list of DUTs
+       rand_one_dut_hostname (pytest fixture): DUT hostname
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    duthost.shell('sudo pfcwd start_default')
+    yield
+
+    """ teardown code """
+    duthost.shell('sudo pfcwd stop')
```

Comment:
With these func scoped fixture, at the end of last test run, pfcwd will be stopped and it might affect subsequent tests from other modules. So add another module scoped fixture like the one below, which will ensure that pfcwd is started at the end of the module.

def  func():
yield
duthost.shell('pfcwd start_default')